src/Specific/index.js

```diff
@@ -85,9 +87,9 @@ class Specific {
             return item.every((item) => typeof item === 'string');
         }
 
-        if (item === null || typeof item === 'string') { return true; }
+        if (typeof item === 'string') { return true; }
 
-        if (typeof item === 'object' && (
+        if (item !== null && typeof item === 'object' && (
```

Comment:
why not to add `if (!item) { return false; }` at the start of the function? (It will handle the case of `item = empty string` too)

Reply:
Empty strings and arrays are valid `Specific` values and should return true IMO. Note that we are actually throwing and `false`

```js
if (!Specific.is(specific)) {
        throw runtimeError(Errors.UNEXPECTED_ARGUMENT_SPECIFIC, typeof specific);
}
```